providers/fips-sources.checksums

```diff
@@ -1,459 +0,0 @@
-0e22ea0cf34ef3871e30df0bc302dc29352d38001d1622ddb78a27a374b6aee8  crypto/aes/aes_cbc.c
```

Comment:
Did you really mean to delete the checksums?

Reply:
No I didn't.  They were wrong but not being rebuilt.  I thought I'd updated them.